recipes/box2d/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.4)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
```

```suggestion
add_executable(test_package ../test_package/test_package.cpp)
```

and you can remove test_package.cpp under test_v1_package, to avoid code duplication.